server/lib/configuration.js

```diff
@@ -494,6 +501,12 @@ var conf = module.exports = convict({
     },
     sample_rate: 1
   },
+  sync_tokenserver_url: {
+    default: 'http://127.0.0.1:5000/token',
```

Comment:
> I thought we decided all URLs should be the prod URLs

I think the decision was both ways but we decided to have local urls. I think it was to make sure prod was properly configured.